pipenv/utils.py

```diff
@@ -442,6 +445,7 @@ def resolve_deps(
             collected_hashes = []
             if any('python.org' in source['url'] or 'pypi.org' in source['url']
                    for source in sources):
+                #FIXME Add support for mirrors.
```

Comment:
Is this still relevant?

Reply:
Nope, this was a note-to-self that was missed in my pre-commit diff grep. Removed.

Reply:
That being said, it looks like something might have been missed with venv_resolve_deps, I'll take a look now.

Reply:
Fixed the missing bit, will push shortly.